Question: Unable to compile - "unknown type name ‘bool’; did you mean ‘_Bool’?"

Hello, 

any idea what dependency am I missing?
I am unable to compile hello world example because of following errors:
 
```
   Compiling ext-php-rs v0.7.3
The following warnings were emitted during compilation:

warning: In file included from src/wrapper.c:1:0:
warning: src/wrapper.h:7:71: error: unknown type name ‘bool’; did you mean ‘_Bool’?
warning:  zend_string *ext_php_rs_zend_string_init(const char *str, size_t len, bool persistent);
warning:                                                                        ^~~~
warning:                                                                        _Bool
warning: src/wrapper.c:3:71: error: unknown type name ‘bool’; did you mean ‘_Bool’?
warning:  zend_string *ext_php_rs_zend_string_init(const char *str, size_t len, bool persistent)
warning:                                                                        ^~~~
warning:                                                                        _Bool

error: failed to run custom build command for `ext-php-rs v0.7.3`

Caused by:
  process didn't exit successfully: `/home/mrceperka/projects/mrceperka/ext-php-rs-example/target/debug/build/ext-php-rs-7bf2e69a5b04eccc/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-changed=src/wrapper.h
  cargo:rerun-if-changed=src/wrapper.c
  cargo:rerun-if-changed=allowed_bindings.rs
  TARGET = Some("x86_64-unknown-linux-gnu")
  OPT_LEVEL = Some("0")
  HOST = Some("x86_64-unknown-linux-gnu")
  CC_x86_64-unknown-linux-gnu = None
  CC_x86_64_unknown_linux_gnu = None
  HOST_CC = None
  CC = None
  CFLAGS_x86_64-unknown-linux-gnu = None
  CFLAGS_x86_64_unknown_linux_gnu = None
  HOST_CFLAGS = None
  CFLAGS = None
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some("true")
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  running: "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-m64" "-I" "/usr/include/php/20180731" "-I" "/usr/include/php/20180731/main" "-I" "/usr/include/php/20180731/TSRM" "-I" "/usr/include/php/20180731/Zend" "-I" "/usr/include/php/20180731/ext" "-I" "/usr/include/php/20180731/ext/date/lib\n" "-Wall" "-Wextra" "-o" "/home/mrceperka/projects/mrceperka/ext-php-rs-example/target/debug/build/ext-php-rs-adeee84e1b8aaa1e/out/src/wrapper.o" "-c" "src/wrapper.c"
  cargo:warning=In file included from src/wrapper.c:1:0:
  cargo:warning=src/wrapper.h:7:71: error: unknown type name ‘bool’; did you mean ‘_Bool’?
  cargo:warning= zend_string *ext_php_rs_zend_string_init(const char *str, size_t len, bool persistent);
  cargo:warning=                                                                       ^~~~
  cargo:warning=                                                                       _Bool
  cargo:warning=src/wrapper.c:3:71: error: unknown type name ‘bool’; did you mean ‘_Bool’?
  cargo:warning= zend_string *ext_php_rs_zend_string_init(const char *str, size_t len, bool persistent)
  cargo:warning=                                                                       ^~~~
  cargo:warning=                                                                       _Bool
  exit status: 1

  --- stderr


  error occurred: Command "cc" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-m64" "-I" "/usr/include/php/20180731" "-I" "/usr/include/php/20180731/main" "-I" "/usr/include/php/20180731/TSRM" "-I" "/usr/include/php/20180731/Zend" "-I" "/usr/include/php/20180731/ext" "-I" "/usr/include/php/20180731/ext/date/lib\n" "-Wall" "-Wextra" "-o" "/home/mrceperka/projects/mrceperka/ext-php-rs-example/target/debug/build/ext-php-rs-adeee84e1b8aaa1e/out/src/wrapper.o" "-c" "src/wrapper.c" with args "cc" did not execute successfully (status code exit status: 1).
  ```


Platform: 
- Ubuntu 18.04
- cc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
- rustc 1.56.1


Thanks :pray:

--- a/src/args.rs
+++ b/src/args.rs
@@ -168,6 +168,7 @@ impl<'a> Arg<'a> {
             )
             .ok_or(Error::InvalidCString)?,
             default_value: match &self.default_value {
+                Some(val) if val.as_str() == "None" => CString::new("null")?.into_raw(),
                 Some(val) => CString::new(val.as_str())?.into_raw(),
                 None => ptr::null(),
             },

--- a/tests/src/integration/defaults/defaults.php
+++ b/tests/src/integration/defaults/defaults.php
@@ -4,3 +4,6 @@ assert(test_defaults_integer() === 42);
 assert(test_defaults_integer(12) === 12);
 assert(test_defaults_nullable_string() === null);
 assert(test_defaults_nullable_string('test') === 'test');
+assert(test_defaults_multiple_option_arguments() === 'Default');
+assert(test_defaults_multiple_option_arguments(a: 'a') === 'a');
+assert(test_defaults_multiple_option_arguments(b: 'b') === 'b');

--- a/tests/src/integration/defaults/mod.rs
+++ b/tests/src/integration/defaults/mod.rs
@@ -12,10 +12,20 @@ pub fn test_defaults_nullable_string(a: Option<String>) -> Option<String> {
     a
 }
 
+#[php_function]
+#[php(defaults(a = None, b = None))]
+pub fn test_defaults_multiple_option_arguments(
+    a: Option<String>,
+    b: Option<String>,
+) -> PhpResult<String> {
+    Ok(a.or(b).unwrap_or_else(|| "Default".to_string()))
+}
+
 pub fn build_module(builder: ModuleBuilder) -> ModuleBuilder {
     builder
         .function(wrap_function!(test_defaults_integer))
         .function(wrap_function!(test_defaults_nullable_string))
+        .function(wrap_function!(test_defaults_multiple_option_arguments))
 }
 
 #[cfg(test)]

